### PR TITLE
Need to add getBlockDevices function

### PR DESCRIPTION
**Pull request by mattcui**
For addressing [Need to check if EphemeralDisk is attached within a timeout value](https://www.pivotaltracker.com/story/show/134949679) story from `softlayer-go` side, I will submit another PR to `bosh-softlayer-cpi` once this PR is merged. 

@maximilien please review and merge, thanks.

### CODE REVIEW

File: services/softlayer_virtual_guest.go [hunk 1]
```diff
@@ -1200,6 +1200,21 @@ func (slvgs *softLayer_Virtual_Guest_Service) GetLocalDiskFlag(instanceId int) (
 	return false, errors.New(fmt.Sprintf("Failed to check the disk type (local or SAN) of that virtual guest with id '%d', got '%s' as response from the API.", instanceId, res))
 }
 
+func (slvgs *softLayer_Virtual_Guest_Service) GetBlockDevices(instanceId int) ([]datatypes.SoftLayer_Virtual_Guest_Block_Device, error) {
+	response, _, err := slvgs.client.GetHttpClient().DoRawHttpRequest(fmt.Sprintf("%s/%d/getBlockDevices.json", slvgs.GetName(), instanceId), "GET", new(bytes.Buffer))
```
Comment (maximilien) [inline services/softlayer_virtual_guest.go]: Why are you ignoring the second return value?

Comment (maximilien) [inline services/softlayer_virtual_guest.go]: See how I do the other methods and do the same.

File: services/softlayer_virtual_guest_test.go [hunk 2]
```diff
@@ -2238,4 +2238,36 @@ var _ = Describe("SoftLayer_Virtual_Guest_Service", func() {
 			})
 		})
 	})
+
+	Context("#GetBlockDevices", func() {
```
Comment (maximilien) [inline services/softlayer_virtual_guest_test.go]: You'll also need to add a test for HTTP errors.

File: services/softlayer_virtual_guest.go (hunk 1 above)
Comment (mattcui) [inline services/softlayer_virtual_guest.go]: Due to [PR 136](https://github.com/maximilien/softlayer-go/pull/136), http return code has already been checked in [http_client.go](https://github.com/bondyk/softlayer-go/blob/abaec9812b2470f1f13b88eb06714fc280b88c88/client/http_client.go#L210), so if the http return code > 400, http client will return an error, there is no need to check http code in `GetBlockDevices` and all alike functions.

File: services/softlayer_virtual_guest_test.go (hunk 2 above)
Comment (mattcui) [inline services/softlayer_virtual_guest_test.go]: Please see above explanation why there is no need to check http return code.

**Comment by mattcui** [comment]
@maximilien I replied your comments, please let me know if you still have questions. Thanks.

**Pull request merged**